workflows/viralrecon.nf

```diff
@@ -1,141 +0,0 @@
-/*
-========================================================================================
```

Comment:
Where is the code that was on `viralrecon.nf` now?

Reply:
This came in via the template sync but we have two separate workflows to process `illumina` and `nanopore` data so we don't need one named by the pipeline itself which is why I just deleted it.